applications/DeepSpeed-Chat/training/step3_rlhf_finetuning/main.py

```diff
@@ -514,16 +522,23 @@ def main():
                     random.shuffle(exp_dataset)
                     random.shuffle(unsup_dataset)
 
+                end = time.time()
+
                 print_rank_0(
-                    f'epoch: {epoch}|step: {step}|ppo_ep: {ppo_ep+1}|act_loss: {actor_loss_sum/inner_iter}|cri_loss: {critic_loss_sum/inner_iter}|unsuper_loss: {unsup_loss_sum/inner_iter}',
+                    f'Epoch: {epoch} | Step: {step} | PPO Epoch: {ppo_ep+1} | Actor Loss: {actor_loss_sum/inner_iter} | Critic Loss: {critic_loss_sum/inner_iter} | Unsupervised Loss: {unsup_loss_sum/inner_iter}',
                     args.global_rank)
+                print_throughput(rlhf_engine.actor.model, args, end - start,
```

Comment:
Nit: Can these calculations be moved inside the `print_throughput` function? E.g. `end -start`, `training_start - generate_start`, etc? The args will just become the various recorded times and the function will compute the deltas internally.